src/app/examples/segmented-chip-control-example/segmented-chip-control-example.component.html

```diff
@@ -0,0 +1,6 @@
+<kirby-segmented-chip-control [items]="items" (chipSelect)="onChipSelected($event)" [selectedItem]="selectedItem">
```

Comment:
Use naming convention:
```suggestion
<kirby-segmented-chip-control [items]="items" (chipSelect)="onChipSelect($event)" [selectedItem]="selectedItem">
```

Reply:
Done